Parsers/pfSense/syslogng-pfsense.txt

```diff
@@ -0,0 +1,67 @@
+// pfSense Syslog-NG Parsing Rule
+// Last Updated Date: March 09, 20211
+//
+// Syslog-NG Parsing Rule
+// https://docs.netgate.com/pfsense/en/latest/monitoring/logs/raw-filter-format.html​​​​​​​
+//
+// Usage Instruction :
+// Extract rule descriptions with associated tracking number (Optional & pfSense Only)
+// In pfSense and go to diagnostics -> Command Prompt
+// Enter one of the following command in the execute shell command box and click the execute button
+//
+// pfctl -vv -sr | grep label | sed -r 's/@([[:digit:]]+).*(label "|label "USER_RULE: )(.*)".*/"\1","\3"/g' | sort -V -u | awk 'NR==1{$0="\"Rule\",\"Label\""RS$0}7'
```

Comment:
Is this a sample event that would show up in the table?  It looks like a command line to execute.  We need an event sample and I don't see any Sample data for pfSense - see step 4 from here - https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/ReadMe.md#build-the-connector.